pkg/common-controller/snapshot_controller.go

```diff
@@ -96,17 +96,6 @@ func (ctrl *csiSnapshotCommonController) syncContent(content *crdv1.VolumeSnapsh
 		return err
 	}
 
-	// TODO(xiangqian): Putting this check in controller as webhook has not been implemented
```

Comment:
I'm still in favor of keeping this check as the validation webhook may not be deployed.  @yuxiangqian what do you think?

Reply:
I agree with this for backwards compatibility.

Reply:
agreed. Let's leave it there but update the. comments?

Reply:
done!